drake/systems/frames/CoordinateFrame.m

```diff
@@ -1,20 +1,28 @@
 classdef CoordinateFrame < handle
-% Every input, state, and output in a DynamicalSystem has a coordinate frame
-% attached to it.  Many bugs can be avoided by forcing developers to be
-% explicit about these coordinate systems when they make combinations of
-% systems.
-
+  % Every input, state, and output in a DynamicalSystem has a coordinate frame
+  % attached to it.  Many bugs can be avoided by forcing developers to be
+  % explicit about these coordinate systems when they make combinations of
+  % systems.
+  
   properties (SetAccess=private,GetAccess=public)
     name='';        % string name for this coordinate system
     dim=0;          % scalar dimension of this coordinate system
     transforms={};  % handles to CoordinateTransform objects
-    prefix;         % a vector character prefix used for the msspoly variables, or a vector of size dim listing prefixes for each variable
+    prefix;         % a vector character prefix used for the
+                    % msspoly variables, or a vector of size dim
+                    % listing prefixes for each variable
   end
   properties (Access=private)
-    coordinates={}; % list of coordinate names
+    coordinates={}; % list of coordinate names. must be kept
+                    % private to ensure that lazy generation is
+                    % safe. 
+                    % WARNING: since coordinates are generated
+                    % lazily, never access coordinates
+                    % directly even from within the class. 
+                    % Use getCoordinateNames instead.
     poly=[];        % optional msspoly variables for this frame
   end
-
+  
```

Comment:
thank you for pointing that out - I didn't fully understand why there was so much whitespace. I'll fix that so that the diff is easier to read.